README.md

```diff
@@ -1,14 +1,22 @@
 # Project
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This utility can be used to detect the supported TLS version for a given CosmosDB account.
 
-As the maintainer of this project, please make a few updates:
+Note that this utility would only be able to test the account against TLS versions supported by the client system running the script. Most modern operating systems no longer support SSL V2 and SSL V3.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Supported TLS version:
+- SSL V2
```

Comment:
remove the ssl version :)

Reply:
Do you want me to remove SSL V2 and SSL V3 support from the script as well or just the documentation? Currently we have
```
    ssl_versions.append(("SSL V2", ssl.PROTOCOL_SSLv2 if ssl.HAS_SSLv2 else None))
    ssl_versions.append(("SSL V3", ssl.PROTOCOL_SSLv3 if ssl.HAS_SSLv3 else None))
```

Reply:
@j-patrick for my understanding, why should we remove the SSL version?

Reply:
Since we do not support sslv2 or v3, I think we should just get rid of it from this tool. thoughts?

Reply:
We do support SSL3 don't we?

Reply:
Removed SSL V2 and V3. @ThomasWeiss, Justin mentioned the main reason was that we anyway by default don't support SSL V2 and V3 in cosmosdb so there's no point of having these checked by the script.